lib/applications/resolvers.ts

```diff
@@ -28,7 +32,12 @@ export const createApplication: Resolver = async (_, args, { prisma }) => {
   let application;
   try {
     application = await prisma.application.create({
-      data: { ...args.input },
+      data: {
+        ...args.input,
+        applicant: {
+          connect: { id: applicantId },
```

Comment:
Should we also be doing a `connect` like this in `lib/permits/resolvers`? 

Also, there's some TODOs in `lib/permits/schema.graphql` regarding the same task as this PR. I'm not totally sure if your changes already fix them or not but I think it would make sense for you to address them in this pr.

Reply:
Yes we should have a `connect` when creating permits. We always want a created permit to be attached to an applicant.